modules/poetry.nix

```diff
@@ -19,9 +21,78 @@
   }: let
     args = {inherit self system lib config name kernelName requiredRuntimePackages;};
     kernelModule = import ./kernel.nix args;
+
+    kernelFunc = {
```

Comment:
we only keep some necessary args in the `KernelFunc`. (simplify it again) 
 https://github.com/GTrunSec/jupyterWith/blob/dev/kernels/available/python/default.nix#L7

Reply:
fixes #485